src/Parsers/ParserCreateFunctionQuery.cpp

```diff
@@ -25,6 +32,16 @@ bool ParserCreateFunctionQuery::parseImpl(IParser::Pos & pos, ASTPtr & node, Exp
     ParserKeyword s_aggr_function("AGGREGATE FUNCTION");
     ParserKeyword s_returns("RETURNS");
     ParserKeyword s_javascript_type("LANGUAGE JAVASCRIPT");
+    ParserKeyword s_type_remote("TYPE Remote");
```

Comment:
I want tot confirm whether in the DDL, the type has to be `Remote`, cannot be `REMOTE` or `remote`.

Reply:
Keyword is case insensitive

Reply:
in my opinion, I prefer use syntax `CREATE REMOTE FUNCTION ...` instead of `CREAET FUNCTION ... TYPE REMOTE`

Reply:
I agree "CREATE REMOTE FUNCTION" is better then "CREAET FUNCTION ... TYPE REMOTE"